libs/coin-evm/package.json

```diff
@@ -63,12 +63,14 @@
     "eip55": "^2.1.1",
     "ethers": "5.7.2",
     "expect": "^27.4.6",
+    "imurmurhash": "^0.1.4",
```

Comment:
The last commit on this lib was 10+ years ago and the repo has very few stars, don't you think this would be a "better" alternative: https://github.com/karanlyons/murmurHash3.js

Reply:
Nope, this is the most used version and more specifically the fastest implementation by far. I already benched the murmurhash libs, and this one should be the best option 👍

Reply:
(In this _almost_ cryptography world, implementations are pretty static once built & working. You have a spec, you follow it to get your implementation, then you probably won't touch it except for an important bugfix)

Reply:
For example here, all the libs you'll find are just rewritings of:
- https://github.com/garycourt/murmurhash-js
- https://github.com/kazuyukitanimura/murmurhash-js (which is a fork of the gaycourt repo)

And both are 12 years old repo, but fully functional.